src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/config/WebClientConfiguration.kt

```diff
@@ -26,93 +26,59 @@ class WebClientConfiguration(
   @param:Value("\${hmpps.govuk.api.url}") private val govUkApiUrl: String,
   @param:Value("\${hmpps.hdc.api.url}") private val hdcApiUrl: String,
   @param:Value("\${os.places.api.url}") private val osPlacesApiUrl: String,
+  @param:Value("\${retry.timeout:60s}") val timeoutSeconds: Duration,
```

Comment:
60 seconds, is this too much?

Reply:
is this timeout on any request? or just when getting a token?

If just a token then we probably want it down to like 5 seconds or something 

We could see what other teams use

Reply:
It's on the token,.. I've seen a lot of 20s,... I will change it to 10